pwiz_tools/Skyline/Menus/ViewMenu.cs

```diff
@@ -827,6 +832,7 @@ public void DocumentUiChanged()
         }
         public void ViewMenuDropDownOpening()
         {
+            viewTargetsMenuItem.Checked = Settings.Default.ShowPeptides;
```

Comment:
It looks like the targets window can be shown without setting Settings.Default.ShowPeptides to true, which should get fixed before relying on it in this way. Look for uses of SkylineWindow.ShowSequenceTreeForm().

Reply:
Huh, ShowSequenceTreeForm(false) appears to very carefully not change the value of Settings.Default.ShowPeptides, so I guess I don't really understand what that value means, but it seems it's not a reliable indicator of that window's visibility.

I'll set that checkmark value to reflect the targets window being non-null and visible.

Reply:
Sounds safest. I am pretty sure we are not either hooking when the user clicks the X button to close the Targets window. So, yeah, not reliable to use the Settings value as an indicator of Targets view visibility.